.githooks/prepare-commit-msg/prepend-branch-id.js

```diff
@@ -39,19 +41,24 @@ if (/COMMIT_EDITMSG/g.test(commitFile)) {
 
           // '(no branch)' indicates we are in a rebase or other non-HEAD scenario
           if (name !== '(no branch)') {
-            const branchId = utils.getBranchId(name)
-            const contentsStartWithValidBranchId = utils.startsWithValidBranchId(contents.toString())
+            const branchId = utils.getBranchId(name);
+            const contentsStartWithValidBranchId = utils.startsWithValidBranchId(contents.toString());
+
+            console.log('branchId',branchId ); // TODO remove me
+            console.log('contentsStartWithValidBranchId',contentsStartWithValidBranchId ); // TODO remove me
```

Comment:
TODO? 😉

Reply:
I used it for testing and I'm sure you could work out what these bits were doing, but it left it so you could see in the console.